maps/Sudan/Sudan_conflict_points

```diff
@@ -0,0 +1,85 @@
+// Author: Iman Smith
+// Description:
+//  Script for generating conflict event mask for Sudan from 2020-2024
+
+var roi = ee.FeatureCollection("FAO/GAUL/2015/level0")
+  .filter(ee.Filter.eq('ADM0_NAME', 'Sudan'));
+
+Map.centerObject(roi,5);
+
+var pointVisParams = {
+  color: 'FF0000' // Red color
+};
+
+// Add the CSV points to the map
+Map.addLayer(table, pointVisParams, 'Conflict Points');
+
+var Table = ee.FeatureCollection("projects/ee-ismith17/assets/2020-2024_SUDAN");
```

Comment:
Please rename this from `Table` to `table` move it above line 15. Otherwise the script fails to run from a direct copy and paste